articles/contributing-style-guide.md

```diff
@@ -2,59 +2,58 @@
+In this section, we detail how we meet this obligation. We'll offer explicit guidance that helps the Q# development community.
 
 ### Operations and functions ###
 
-One of the first things that a name should establish is whether a given symbol represents a function or an operation.
-The difference between functions and operations is critical to understanding how a block of code behaves.
-To communicate the distinction between functions and operations to users, we rely on that Q# models quantum operations through the use of side effects.
+Our name choices should first establish whether a symbol represents a function or an operation.
+The difference between functions and operations indicates how a block of code behaves.
+We rely on Q# models quantum operations to communicate the difference between functions and operations to users, through the use of side effects.
 That is, an operation *does* something.
 
-By contrast, functions describe the mathematical relationships between data.
+In contrast, functions describe the mathematical relationships between data.
 The expression `Sin(PI() / 2.0)` *is* `1.0`, and implies nothing about the state of a program or its qubits.
 
-Summarizing, operations do things while functions are things.
-This distinction suggests that we name operations as verbs and functions as nouns.
+In summary, operations do things, while functions are things.
+Based on this difference, we name operations as verbs and functions as nouns.
 
 > [!NOTE]
 > When a user-defined type is declared, a new function that constructs instances of that type is implicitly defined at the same time.
-> From that perspective, user-defined types should be named as nouns so that both the type itself and the constructor function have consistent names.
+> From that perspective, user-defined types should be nouns so that both the type itself and the constructor function have consistent names.
 
-Where reasonable, ensure that operation names begin with verbs that clearly indicate the effect taken by the operation.
+Ensure that operation names begin with verbs that clearly indicate the effect taken by the operation.
 For example:
 
 - `MeasureInteger`
 - `EstimateEnergy`
 - `SampleInt`
 
-One case that deserves special mention is when an operation takes another operation as input and calls it.
-In such cases, the action taken by the input operation is not clear when the outer operation is defined, such that the right verb is not immediately clear.
+We should pay special attention when an operation takes another operation as input. Also, when the operation calls it.
```

Comment:
I think this edit changes the meaning of the original text, making it sound like "takes another operation" and "calls it" are two different cases rather than two steps in a single case.